CHANGELOG.rst

```diff
@@ -2,6 +2,8 @@
 Release Notes
 =============
 
+.. current developments
```

Comment:
We want to ensure that we want to insert new news under the ```.. current developments``` section in all cookiecuttered projects (currently implemented snmf, utils, structures, srmise, etc).